lib/client.js

```diff
@@ -1331,6 +1331,65 @@ MatrixClient.prototype.invite = function(roomId, userId, callback) {
 };
 
 /**
+ * Invite a user to a room based on their email address.
+ * @param {string} roomId The room to invite the user to.
+ * @param {string} email The email address to invite.
+ * @param {string} displayName The display name to use. If omitted, everything
```

Comment:
displayName should probably be removed entirely, given our discussion today, no?